### DISCUSSION TITLE
[resolved] debian 环境下使用 libtquic.a 时链接错误：undefined reference cbrt

### DISCUSSION THREAD

**Discussion by nlifew** [Tencent/tquic#370] [General]
正常构建 tquic 后得到 libtquic.a。集成到 c/c++ 项目中以后链接报错，cbrt 符号未定义。

解决方案：在 c/c++ 项目里添加 -lm 选项以链接 libm 库

我并没有在 issue 区看到过类似提问，但还是想把解决方案记录下来。这不是一个 bug report，也不是 feature request 或 security vulnerability，因此我没有使用相关模板。我想知道您对 pr 的态度。如果您认为合适，我会提交一个 pr 到 tquic-website 仓库的”常见问题“下。对于使用 c/c++的工程师，这可能会对他们有所帮助。

谢谢

**Comment by iyangsj** [comment]
欢迎提交 PR 到 tquic-website 仓库的”常见问题“下！
